simulation_server.py

```diff
@@ -285,15 +285,15 @@ def runWebotsInThread(client):
                     envVarDocker["DISPLAY"] = display
                     envVarDocker["XAUTHORITY"] = xauth
 
-                config['dockerConfDir'] = 'config/simulation/docker'
+                config['dockerConfDir'] = os.path.abspath(os.path.dirname( __file__)) + '/config/simulation/docker'
                 # create a Dockerfile if not provided in the project folder
                 defaultDockerfilePath = ''
                 if not os.path.isfile('Dockerfile'):
                     defaultDockerfilePath = config['dockerConfDir'] + '/Dockerfile.default'
                     if os.path.exists(defaultDockerfilePath):
                         os.system(f'ln -s {defaultDockerfilePath} ./Dockerfile')
                     else:
-                        error = f"error: Missing Dockerfile.default in {config['dockerConfDir']}"
+                        error = f"error: Missing Dockerfile.default in {defaultDockerfilePath}"
```

Comment:
I wouldn't change this line as we want to refer to the folder, not the file:
```suggestion
                        error = f"error: Missing Dockerfile.default in {config['dockerConfDir']}"
```